src/lib/minions/functions/addSkillingClueToLoot.ts

```diff
@@ -23,6 +23,9 @@ export default function addSkillingClueToLoot(
 	const userLevel = user.skillLevel(skill);
 	const chance = Math.floor(clueChance / (100 + userLevel));
 	let nests = 0;
+	let cluesTotalWeight = 0;
+	clues.forEach(c => (cluesTotalWeight += c[1]));
```

Comment:
no `forEach`! your most convenient way to do this is something like: `addArrayOfNumbers(clues.map(c => c[1])`